barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -70,16 +70,15 @@ void ProtoGalaxyVerifier_<VerifierInstances>::receive_and_finalise_instance(cons
     inst->log_instance_size = static_cast<size_t>(numeric::get_msb(inst->instance_size));
     inst->public_input_size =
         transcript->template receive_from_prover<uint32_t>(domain_separator + "_public_input_size");
+    inst->pub_inputs_offset =
```

Comment:
moved above the public inputs to align with ultra_prover's ordering of what it sends